cmd/modern/root/install/mssql.go

```diff
@@ -29,6 +29,9 @@ func (c *Mssql) DefineCommand(...cmdparser.CommandOptions) {
 			{
 				Description: "Create SQL Server, download and attach AdventureWorks sample database",
 				Steps:       []string{"sqlcmd create mssql --using https://aka.ms/AdventureWorksLT.bak"}},
+			{
+				Description: "Create SQL Server, download and attach AdventureWorks sample database with different database name",
+				Steps:       []string{"sqlcmd create mssql --using https://aka.ms/AdventureWorksLT.bak,MyDatabaseName"}},
```

Comment:
Let's make the example:

--using [https://aka.ms/AdventureWorksLT.bak,adventureworks](https://aka.ms/AdventureWorksLT.bak,adventureworks)

All the system databases are lower case, to customers tend to lean towards lower case (snake case for spacing).